### ISSUE TITLE
Label file-size related `th` with class name

### ISSUE THREAD

**Issue by qooob** [webmin/webmin#194]
It would be a good idea if all `table > th` (speaking about table headers), that contain Quota or other size related data, could have a class `.size-related-data` or any other. In this case, I will be able to initialize `dataTables` and make it properly sort size (TB, GB, MB, KB and bytes) data!

Could you do it?

**Comment by jcameron** [comment]
In theory, yes ... however, there are a LOT of places where such tables exist in Webmin.

Perhaps another approach would be for the sorting code to look at the table values for numeric or numerics with units like KB, and if they are all like this to sort appropriately.

**Comment by swelljoe** [comment]
That method is somewhat error prone, and there's a surprising number of differences in how Webmin outputs stuff like this. Not to say it might not be the expedient path, but in the long run it might be worth having the data structure being passed to the ui_ code able to indicate what kind of data it contains, from a sorting and display perspective.

**Comment by qooob** [comment]
Alright, let's not do this then and I will try to do it the other way, like detecting columns using (TB, GB, MB, KB and bytes) data.

Let me further explain. Table should have `thead->th`. That `th` should be the _title_ for each column. That is all, nothing else. For now, far not all tables are made like that. This is why I just can't initialize it on all tables but have to selectively say, where it's ok.

The way it was fixed in Quotas (I just looked at it yesterday - will not work as well). Let me comment there in a moment.
